pkgs/applications/misc/gnuradio/default.nix

```diff
@@ -41,8 +41,14 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  patchPhase = ''
```

Comment:
Great. It's *generally* better to use `postPatch` instead of overriding the entire patch phase, on the off change that somebody wanted to easily apply other patches by overrding `patches`.

Reply:
OK, updated